document/field_boolean.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/blevesearch/bleve/analysis"
 )
 
-const DefaultBooleanIndexingOptions = StoreField | IndexField
+const DefaultBooleanIndexingOptions = StoreField | IndexField | IncludeDocValues
```

Comment:
I thought we were going to go with the shorter "DocValues".

edit: yes I see later you use just "DocValues" in the mapping.  Is there any reason we can't use the same everywhere, otherwise we have to remember which one to use where.